xocto/localtime.py

```diff
@@ -60,12 +60,15 @@ def datetime(
     minute: int = 0,
     second: int = 0,
     microsecond: int = 0,
+    tz: timezone.zoneinfo.ZoneInfo | None = None,
```

Comment:
```suggestion
    tz: zoneinfo.ZoneInfo | None = None,
```

Due to some refactoring, the leading `timezone` is causing mypy indigestion. We get the type directly from the `zoneinfo` stdlib now.